ops/packer/build_bfd-server.json

```diff
@@ -34,7 +34,7 @@
       "type": "ansible",
       "playbook_file": "./build_bfd-server.yml",
       "extra_arguments": [
-        "-e ansible_python_interpreter=/usr/bin/python",
+        "-e ansible_python_interpreter=/usr/bin/python3",
```

Comment:
OK! I think this may work fine, actually. The base AMIs appear to include `python3.x86_64 3.7.10-1.amzn2.0.1` maintaining the following:
```
/usr/bin/pydoc3
/usr/bin/pydoc3.7
/usr/bin/python3
/usr/bin/python3.7
/usr/bin/python3.7m
/usr/bin/pyvenv
/usr/bin/pyvenv-3.7
/usr/share/doc/python3-3.7.10
/usr/share/doc/python3-3.7.10/README.rst
/usr/share/licenses/python3-3.7.10
/usr/share/licenses/python3-3.7.10/LICENSE
/usr/share/man/man1/python3.1.gz
/usr/share/man/man1/python3.7.1.gz
```
I was misled by a default `$PATH` variable that prioritizes `/bin` over `/usr/bin`

As it stands, I think we're ready to test this in Jenkins.

Reply:
Building test succeeded after executing Ansible script